lighthouse-core/lib/file-namer.js

```diff
@@ -20,7 +20,7 @@
  * @return {string}
  */
 function getFilenamePrefix(lhr) {
-  const hostname = new (getUrlConstructor())(lhr.finalUrl).hostname;
+  const hostname = new URL(lhr.finalUrl).hostname;
```

Comment:
any particular reason this was done?

Reply:
@brendankenny upgraded us to node10+ which now has URL built in so this simplification was possible.